### PR TITLE
fix spelling mistake and optimize code examples

### PR DESCRIPTION
**Pull request by t4k**

### CODE REVIEW

**Comment by shzdp** [comment]
Looks good

File: _episodes/01-advanced-scripting.md [hunk 1]
```diff
@@ -17,14 +17,14 @@ keypoints:
 ## Conditionals
 
 You can use conditional statements to test whether something is true or false
-and do a programatic behavior as a result. Let's go into the molecules
+and do a programmatic behavior as a result. Let's go into the molecules
```
Comment (tmorrell) [inline _episodes/01-advanced-scripting.md:20]: This looks good!

File: _episodes/01-advanced-scripting.md [hunk 2]
```diff
@@ -17,14 +17,14 @@ keypoints:
...
 directory and make a script that will show us molecules with at least a certain number
 of lines. Make a new script called is_big.sh
 
 We know that wc -l gives us the number of lines in a file. Let's save that to a
 variable.
 ~~~
-num=`wc -l $1`
+num=$(wc -l $1)
```
Comment (tmorrell) [inline _episodes/01-advanced-scripting.md:27]: Much better

**Review by tmorrell — APPROVED** [review]
Looks good

**Pull request merged**